k-distribution/include/ocaml/prelude.ml

```diff
@@ -9,6 +9,8 @@ type k = K.t
 exception Stuck of k
 exception Not_implemented
 
+let interned_bottom = [Bottom]
```

Comment:
this contains some code to speed up serialization and deserialization in ocaml also